File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-04a_C.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    const a: u32 = 0;
    const b: u32 = 1;
    const s: u32 = a - b;

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=SUB:{d}\n", .{s}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-04a_C.rs
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let a: u32 = 0;
    let b: u32 = 1;
    let v: u32 = a - b;

    writeln!(out, "OBS=SUB:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
